homeassistant/components/niko_home_control/light.py

```diff
@@ -73,6 +82,12 @@ def name(self):
         """Return the display name of this light."""
         return self._name
 
+    @property
+    def brightness(self):
+        """Return the brightness of the light."""
+        value = self._data.get_brightness(self._light.id)
+        return None if value == 0 else value
```

Comment:
I don't think this property is needed.
IMO, brightness should be calculated inside `async_update`, and the automatic attribute `self._attr_brightness` should be used instead.

For example:
```
        await self._data.async_update()
        state = self._data.get_state(self._light.id)
        self._attr_is_on = state != 0
        if ColorMode.BRIGHTNESS in self.supported_color_modes:
            self._attr_brightness = state * 2.55
```